examples/with-yarn-workspaces/now.json

```
@@ -0,0 +1,9 @@
+{
+  "version": 2,
+  "builds": [
+    { "src": "packages/web-app/package.json", "use": "@now/next" }
+  ],
+  "routes": [
+    { "src": "/(.*)", "dest": "packages/web-app/$1" }
```

This route isn't correct -- the application will be broken. 😄 
```suggestion
    { "src": "(.*)", "dest": "packages/web-app$1", "continue": true }
```